adminguide.rst

```diff
@@ -161,6 +161,7 @@ a typical Flux system instance will read all configuration from
 In this guide, separate files will typically be used for clarity, instead
 of adding all configuration tables to a single TOML file.
 
+See also: :core:man5:`flux-config`.
```

Comment:
Oops, sorry, there isn't a `:core:man5:` ref configured at this time. That will have to be added to `conf.py`

Reply:
Sorry, looks like you already caught that!